src/safeds/ml/nn/_input_conversion.py

```diff
@@ -29,6 +29,15 @@ def _data_conversion_fit(self, input_data: FT, batch_size: int, num_of_classes:
     def _data_conversion_predict(self, input_data: PT, batch_size: int) -> DataLoader:
         pass  # pragma: no cover
 
+    @abstractmethod
+    def _set_parameters(
+        self,
+        target_name: str,
+        time_name: str,
+        feature_names: list[str],
```

Comment:
Consider using `**kwargs` as otherwise we have to add every parameter to this function and change this function every time we add a new, different input conversion

Reply:
added